pkg/symbolizer/symbolizer.go

```diff
@@ -89,8 +114,10 @@ func (s *Symbolizer) runSymbolizationCycle(ctx context.Context) {
 			Limit:  s.batchSize,
 			MinKey: prevMaxKey,
 		})
+		s.attempts.Inc()
 		if err != nil {
 			level.Error(s.logger).Log("msg", "failed to fetch unsymbolized locations", "err", err)
+			s.errors.With(prometheus.Labels{"reason": "fetch_unsymbolized_locations"}).Inc()
```

Comment:
If `fetch_unsymbolized_locations` is too verbose, I will try to come up with a shorter name.

Reply:
I think it's fine :+1: